tools/shared/src/main/scala/org/scalajs/core/tools/sourcemap/SourceMapWriter.scala

```diff
@@ -100,10 +115,16 @@ class SourceMapWriter(
     pendingName = nodePosStack.top._2
   }
 
+  def startNode(column: Int, originalPos: Position): Unit = {
+    nodePosStack.push((originalPos, null))
+    startSegment(column, originalPos, null)
+  }
+
   def startNode(column: Int, originalPos: Position,
-      originalName: Option[String] = None): Unit = {
-    nodePosStack.push((originalPos, originalName))
-    startSegment(column, originalPos, originalName)
+      originalName: Option[String]): Unit = {
+    val _originalName = originalName.orNull
```

Comment:
`orNull` takes an implicit evidence identity function and then calls `getOrElse`. Don't use it. Use an `if/else` instead.